datajoint/admin.py

```diff
@@ -41,13 +43,23 @@ def kill(restriction=None, connection=None):  # pragma: no cover
     query = 'SELECT * FROM information_schema.processlist WHERE id <> CONNECTION_ID()' + (
         "" if restriction is None else ' AND (%s)' % restriction)
 
+    safemode = config.get('safemode', True) if safemode is None else True
```

Comment:
If `safemode` is not None, then its value is ignored

Reply:
`    safemode = config.get('safemode', True) if safemode is None else safemode` ?

Reply:
^ went ahead with this update